controllers/scopeinstance_controller.go

```diff
@@ -104,7 +101,9 @@ func (r *ScopeInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 
 	// create required roleBindings and clusterRoleBindings.
+	log.Log.Error(nil, "XXX calling ensureBindings")
```

Comment:
I don't think this should be an error. Can change it to `Log.Info`

Reply:
I'm not sure we even need to log this in general. If we did want to log this I would probably do something more like:
```suggestion
	log.Log.Info("creating (Cluster)RoleBindings")
```

Reply:
@everettraven oops I left my debug XXX logging in :) I'll remove them. Was trying to figure out where the nil pointer was happening